### ISSUE TITLE
Test run failures: fix localisation and rearrange gui's

### ISSUE THREAD

**Issue by robertcv** [biolab/orange3#4608]
**Expected behavior**
Local run tests should not fail.

**Versions:**
Orange master, PyQt5==5.13.0
Python 3.6.9,
Ubuntu 18.04.4 LTS, GNOME 3.28.2

**Additional context**
In the spirit of the recent quest for segfault, I run the whole test suite locally (with `tox -e py`) and found that there are some tests failing:

```
======================================================================
FAIL: test_binning (test_discretize.TestTimeBinning)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/robert/git/orange3/.tox/py/lib/python3.6/site-packages/Orange/preprocess/tests/test_discretize.py", line 62, in test_binning
    615254400])]
AssertionError: Lists differ: [('10[677 chars]['75 jan', 'jul', '76 jan', 'jul', '77 jan', '[538 chars]00])] != [('10[677 chars]['75 Jan', 'Jul', '76 Jan', 'Jul', '77 Jan', '[538 chars]00])]

First differing element 4:
('6 months', ['75 jan', 'jul', '76 jan', 'jul', '77 jan', '[537 chars]400])
('6 months', ['75 Jan', 'Jul', '76 Jan', 'Jul', '77 Jan', '[537 chars]400])

Diff is 2834 characters long. Set self.maxDiff to None to see it.

======================================================================
FAIL: test_minimum_size (Orange.widgets.data.tests.test_owcontinuize.TestOWContinuize)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/robert/git/orange3/.tox/py/lib/python3.6/site-packages/orangewidget/tests/base.py", line 181, in test_minimum_size
    self.check_minimum_size(widget)
  File "/home/robert/git/orange3/.tox/py/lib/python3.6/site-packages/orangewidget/tests/base.py", line 501, in check_minimum_size
    self.assertLess(min_size.height(), 700)
AssertionError: 724 not less than 700

======================================================================
FAIL: test_minimum_size (Orange.widgets.data.tests.test_owimpute.TestOWImpute)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/robert/git/orange3/.tox/py/lib/python3.6/site-packages/orangewidget/tests/base.py", line 181, in test_minimum_size
    self.check_minimum_size(widget)
  File "/home/robert/git/orange3/.tox/py/lib/python3.6/site-packages/orangewidget/tests/base.py", line 501, in check_minimum_size
    self.assertLess(min_size.height(), 700)
AssertionError: 735 not less than 700

======================================================================
FAIL: test_minimum_size (Orange.widgets.data.tests.test_owpivot.TestOWPivot)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/robert/git/orange3/.tox/py/lib/python3.6/site-packages/orangewidget/tests/base.py", line 181, in test_minimum_size
    self.check_minimum_size(widget)
  File "/home/robert/git/orange3/.tox/py/lib/python3.6/site-packages/orangewidget/tests/base.py", line 501, in check_minimum_size
    self.assertLess(min_size.height(), 700)
AssertionError: 724 not less than 700

======================================================================
FAIL: test_minimum_size (Orange.widgets.evaluate.tests.test_owrocanalysis.TestOWROCAnalysis)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/robert/git/orange3/.tox/py/lib/python3.6/site-packages/orangewidget/tests/base.py", line 181, in test_minimum_size
    self.check_minimum_size(widget)
  File "/home/robert/git/orange3/.tox/py/lib/python3.6/site-packages/orangewidget/tests/base.py", line 501, in check_minimum_size
    self.assertLess(min_size.height(), 700)
AssertionError: 726 not less than 700

----------------------------------------------------------------------
```

**Comment by janezd** [comment]
The first test: Can you show the extended diff, so we see where's the problem?

Others errors: These widget may indeed be too large on your system. The same is happenning in github actions. Can you check? If so, we may need to fix the differences between platforms, included these differences in tests, of change widgets.

**Comment by robertcv** [comment]
For the first, the difference is "jan" - "Jan".

**Comment by janezd** [comment]
Only a blind man could miss it. And me.

It should suffice to modify `Orange.preprocess.tests.test_discretizeTestTimeBinning.testbin` to translate returned names of months from locale to English.

**Comment by janezd** [comment]
Results of our discussion regarding the second problem, widget heights:

- [ ]: Pivot: merge the first three boxes into a single box, where box titles would become labels, probably placed above line edits.
- [ ]: ROC Analysis: we squeeze it, but make it scrollable, so the user can get a smaller graph.
- [ ]: Continuize: rearrange into two columns or make it scrollable, whichever looks better
- [ ]: Purge Domain: two columns
- [ ]: Impute: put the radio buttons in the top box in two columns

**Comment by janezd** [comment]
Fixed via #4716 and #4717.
